public/css/svg.css

```diff
@@ -31,7 +47,8 @@
 }
 
 .length-distribution-title {
-  text-align: center;
+  text-align: left;
+  text-transform: capitalize;
```

Comment:
See if h6 doesn't already provide upercasing. If it doesn't either restyle h6 or create a class `caption` to be applied to h6 (or h5).